lib/blueprinter/view.rb

```diff
@@ -4,12 +4,12 @@ module Blueprinter
   class View
     attr_reader :excluded_field_names, :fields, :included_view_names, :name, :transformers, :definition_order
 
-    def initialize(name, fields: {}, included_view_names: [], excluded_view_names: [],transformers: [])
+    def initialize(name, fields: {}, included_view_names: [], excluded_view_names: [], transformers: [])
       @name = name
       @fields = fields
       @included_view_names = included_view_names
       @excluded_field_names = excluded_view_names
-      @transformers =  transformers
+      @transformers = transformers.empty? ? Blueprinter.configuration.default_transformers.clone : transformers
```

Comment:
Had to make a copy with `clone` because otherwise any time any view's transformers was modified, it would modify the `default_transformers` on the config object. (Don't ask me how long that took to track down... 🤯 )

Reply:
Thanks for making these changes :) 
We're much closer, but there's some edge cases in this logic unfortunately not caught by the unit test that we'll also likely want to cover later in the base_spec.rb
This logic should not belong in the initializer because in a view, when we add a transform, it will not override the blueprinter config, but will still add onto it.
Instead, we should keep track of the view-level transforms separately from the config transform, and on access-time is when we should choose which to use.
For example:

`view.rb`
```rb
def initialize(name, fields: {}, included_view_names: [], excluded_view_names: [], transformers: [])
   ...
   @view_transformers = transformers
   ...
end

...

# At access time is when we choose which transformers to use
def transformers
  view_transformers.empty? ? Blueprinter.configuration.default_transformers : view_transformers
end

...

# Need to make sure when we inherit, we only inherit the view-level transformers (don't wan't to duplicate the config-level one each time we inherit)
def inherit(view)
      ...

      view.view_transformers.each do |transformer|
        add_transformer(transformer)
      end
    end
...

def add_transformer(custom_transformer)
    view_transformers << custom_transformer
end
```

This may be able to be cleaned up but hopefully the above snippet shows some of the edge-cases beyond initialization.
1. We should have some tests to ensure that inheritance (I think there may be some specs in base_spec.rb around inheritance) properly inherits the view-level transformers.
2. We should have some tests in base_render_examples to ensure that we properly override transformers when we define a Blueprinter that adds a transform.

Again, thank you so much for your contributions and please don't hesitate to reach out if you need any help on getting this to the finish line 🙂

Reply:
I think I understand. I just pushed a commit that should handle (2) above. (1) looks like it's already being tested in [base_spec.rb:452](https://github.com/procore/blueprinter/blob/master/spec/integrations/base_spec.rb#L452)? Or perhaps I am not understanding what you're asking for there.